lib/pgrel/active_record/store_chain.rb

```diff
@@ -52,6 +52,37 @@ def not(opts = :chain)
         self
       end
 
+      # Query by store values.
+      # Supports array values.
+      #
+      # NOTE: This method uses "@>" (contains) operator with logic (AND/OR)
+      # and not uses "->" (value-by-key). The use of "contains" operator allows us to
+      # use GIN index effectively.
+      #
+      # Example
+      #   Model.create!(name: 'first', store: {b: 1, c: 2})
+      #   Model.create!(name: 'second', store: {b: 2, c: 3})
+      #
+      #   Model.store(:store, c: 2).all #=> [Model(name: 'first', ...)]
+      #   #=> (SQL) select * from ... where store @> '"c"=>"2"'::hstore
+      #
+      #   Model.store(:store, b: [1, 2]).size #=> 2
+      #   #=> (SQL) select * from ... where (store @> '"c"=>"1"'::hstore) or
+      #                                     (store @> '"c"=>"2"'::hstore)
+      def where(opts)
+        update_scope(
+          opts.map do |k, v|
+            case v
+            when Array
+              "(#{build_or_contains(k, v)})"
+            else
+              contains_clause(k => v)
+            end
+          end.join(' and ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.